### PR TITLE
Documentation of AssemblerControlEvent and AssemblerResultEvent

### PR DESCRIPTION
**Pull request by mjadhav1**
Please review the request and suggest the changes.

### CODE REVIEW

File: src/main/java/edu/asu/plp/tool/backend/isa/events/AssemblerResultEvent.java [hunk 1]
```diff
@@ -2,13 +2,24 @@
...
-	
+
+	/**
+	 * This is a constructor for which accepts Success, ErrorMessage, ProjectName and ASMImage parameters
+	 * @param {String} Success        - It takes the assembler success parameter
+	 * @param {String} ErrorMessage   - It takes the error message parameter
+	 * @param {string} ProjectName    - It takes the name of the project
+	 * @param {string} ASMImage       - It takes the ASM Image
+	 * **/
+
```
Comment (tanpuresiddhant) [inline src/main/java/edu/asu/plp/tool/backend/isa/events/AssemblerResultEvent.java]: Please add functions of parameters for example success - possible values are true which means assembled successfully and false means could not assemble or ProjectName indicate the project name for which the assembler result event generated

File: src/main/java/edu/asu/plp/tool/backend/isa/events/AssemblerControlEvent.java [hunk 2]
```diff
@@ -4,32 +4,55 @@
...
+    /**
+     ** This method returns the name of the project
+    **/
 	public String getProjectName() {
 		return projectName;
 	}
 
+	/**
+	 ** This function returns the project type
+    **/
```
Comment (tobielf) [inline src/main/java/edu/asu/plp/tool/backend/isa/events/AssemblerControlEvent.java]: Coding style, please unify the indentation, currently we use tab among all project files.
For JavaDoc block comment is:
```
/**
 *
 *
 */
```

**Comment by mjadhav1** [comment]
I have committed the requested changes. Please review and accept my pull request.

**Comment by tobielf** [comment]
@mjadhav1 You can merge your branch now.

**Pull request merged**